marklogic-data-hub-central/ui/src/config/tooltips.config.js

```diff
@@ -50,6 +50,10 @@ const AdvMapTooltips = {
     'provGranularity': 'The level of detail logged for provenance. Choose *coarse* for the default level or *off* for no provenance logging.',
  }
 
+const AdvCustomTooltips = {
+    'additionalSettings': 'A JSON object containing key-value pairs to pass as parameters to the custom step module.',
```

Comment:
These are technically passed as "options" to the custom step module, right? I don't think we should say "parameters" here, as that term is used in the context of a custom hook module, and it may lead to confusion.